src/data-display/cards/card-list/card-list-item/PCardListItem.vue

```diff
@@ -0,0 +1,173 @@
+<template>
+    <div class="p-card-list-item" :class="{'rounded': rounded}">
+        <div class="content-area">
+            <div v-if="leftIcon" class="left-icon">
+                <p-i :name="leftIcon"
+                     width="2.5rem"
+                     height="2.5rem"
+                />
+            </div>
+            <div class="content">
+                <slot name="content" />
+            </div>
+            <slot name="right-content-area">
+                <div class="desktop-right-content-area">
+                    <template v-for="(rightContentType, index) in state.rightButtonSet">
+                        <div v-if="CARD_BUTTON_ICON_NAME_TYPE[rightContentType] === CARD_BUTTON_ICON_NAME_TYPE.CUSTOM"
+                             :key="`${rightContentType}-${index}`"
+                        >
+                            <slot name="custom-right-content" />
+                        </div>
+                        <p-icon-button v-else
+                                       :key="`${rightContentType}-${index}`"
```

Comment:
MinorMinor thing,,, I think these `key`s (line17, 22, 39, 45) could be duplicated when some refactor is needed 🤔
How about giving more specify `key`?